designer/submodules/packages/components/src/components/rootRoutes.tsx

```diff
@@ -9,7 +9,8 @@ export function RootRoutes(): JSX.Element {
         <Routes>
             <Route index element={<Navigate to="/scenarios" />} />
             <Route path="scenarios" element={<ScenariosView />} />
-            <Route path="components" element={<ComponentsRoutes />} />
+            <Route path="scenarios/table" element={<ScenariosView table />} />
```

Comment:
I think scenarios with table should stay hidden by default? Was it only for testing?